### PR TITLE
Onl 4867 give input components is sensitive config

### PR DESCRIPTION
**Pull request by stavros-tomas**
I added a "isSensitive" prop to the following components.

-ec-input
-ec-inline-input (copy,edit,loading,value text sub-components)
-ec-amount
-ec-currency-input
-ec dropdown
-ec-dropdown-search

This prop will apply our "sensitive class" if it set.

I created a config.file where you can set the class. If you try to use the prop without to set the class will throw an error.

To showcase this on storybook I included an isSensitive knob on the relavant stories and added this plugin https://www.npmjs.com/package/tailwindcss-filters to apply a blur to the input when the knob is true. The blurring doesn't happen on production.

### CODE REVIEW

File: src/components/ec-dropdown/ec-dropdown.spec.js [hunk 1]
```diff
@@ -133,6 +136,11 @@ describe('EcDropdown', () => {
       const wrapper = mountDropdown({ items, isLoading: true });
       expect(wrapper.element).toMatchSnapshot();
     });
+
+    it('should render with a sensitive class when isSensitive prop is set to true when isSensitive prop is set to true', () => {
```
Comment (matallo) [inline src/components/ec-dropdown/ec-dropdown.spec.js:140]: duplicated spec text or (as I see in the component below) you wanted to mention about the nested `isSensitive`?

**Review by matallo — CHANGES_REQUESTED** [review]
Screenshot 2020-09-09 at 15 42 45

something seems to be missing

also, shall we add the sensitive knob to the Storybook stories?

File: postcss.config.js [hunk 2]
```diff
@@ -15,7 +15,10 @@ module.exports = ({ file }) => {
   if (isProd && shouldPurgeFile) {
     // see https://purgecss.com/guides/vue.html for reference
     config.plugins['@fullhuman/postcss-purgecss'] = {
-      content: ['./src/**/*.vue', './src/**/*.story.js'],
+      content: [
+        './src/**/*.vue',
+        './src/**/*.story.js',
+        './.storybook/*.js'],
```
Comment (stavros-tomas) [inline postcss.config.js:21]: @jaimeperezpedra This will change to './.preview/*.js' after you merge your pr

**Review by matallo — APPROVED** [review]

File: postcss.config.js (hunk 2 above)
Comment (jaimeperezpedra) [reply postcss.config.js:21]: the postcss.config.js is not changing, I changed the config.js from the .storybook folder

**Review by jaimeperezpedra — APPROVED** [review]

**Pull request merged**